openmdao/docs/features/building_blocks/drivers/index.rst

```diff
@@ -9,3 +9,4 @@ Drivers
     scipy_optimize_driver.rst
     pyoptsparse_driver.rst
     genetic_algorithm.rst
+    doe_driver.rst
```

Comment:
doe_driver.rst appears to be missing from the PR

Reply:
aw crap